File: Python/Warmup - 2/string_match.py
def string_match(a, b):
    
    count = 0
    
    shorter = min(len(a), len(b))
    
    for i in range (shorter-1):
        
        if a[i:i+2] == b[i:i+2]:
            
            count = count + 1
            
    return count

File: Java/Warmup - 2/stringMatch.java
public int stringMatch(String a, String b) {

    int count = 0;
  
    if(a.length() >= 2 && b.length() >= 2) {
  
        for(int i=0;i<a.length() - 1 && i<b.length() - 1; i++) {
        
            if(a.substring(i,i+2).equals(b.substring(i,i+2))) {
        
                count++;
                
            }

        }
        
    }

    return count;

}